ShipMonitor/FrontierApi.cs

```diff
@@ -247,11 +247,8 @@ public static Module ModuleFromJson(string name, JObject json)
                 module.health = Math.Round(Health);
             }
 
-            // Flag if module has modifications
-            if (json["module"]["modifiers"] != null)
-            {
-                module.modified = true;
-            }
+            // Flag if module has engineering modifications
+            module.modified = json["engineer"] != null ? true : false;
```

Comment:
Unless I'm missing something, `!=` already produces a `bool` so we can elide `? true : false` here. But I'm equally happy to see it written out.

Reply:
lol, you're ,of course, correct... I suppose I thought it read better.